app/ApplicationStart.scala

```diff
@@ -12,5 +12,10 @@ class ApplicationStart @Inject() (config: Config) {
     .locations("filesystem:" + config.getString("migrationLocation"))
     .load()
 
-  flyway.migrate
+  try flyway.migrate
+  catch {
+    case e: Exception =>
+      println(s"The Flyway migrations failed: $e")
```

Comment:
println?
Should be logging.

Reply:
At the moment of this Pull there was no Logging set up. I'll fix it in the logging PR